compiler/ilgen/VirtualMachineOperandStack.cpp

```diff
@@ -97,7 +97,7 @@ VirtualMachineOperandStack::Commit(TR::IlBuilder *b)
        // using Drop to add/delete stack elements.
        TR::IlValue* stack = b->Load("OperandStack_base");
        for (int32_t i = _stackTop; i >= 0; i--) {
-           _stack[i] = b->LoadAt(Element,
+           _stack[i] = b->LoadAt(pElement,
```

Comment:
I would prefer you squash this commit with the previous one, so there isn't a "broken" point in the history.